Character.cpp

```diff
@@ -98,20 +98,28 @@ void Character::attackInternal(Character& other)
             b) your stats are boosted 10%
             c) the initial value of your stats is updated to reflect this boosted stat for the next time you defeat another character.
       */
-        assert(false);
+        hitPoints = (*initialHitPoints * 1.1);
+        armor = (*initialArmorLevel * 1.1);
+        attackDamage = (*initialAttackDamage * 1.1);
+        
+        *initialHitPoints = hitPoints;
+        *initialArmorLevel = armor;
+        *initialAttackDamage = attackDamage;
+
```

Comment:
You didn't do `a)`
When you implement `a)`, you'll see that you're doing the same code 3x but with different variables.  so write a free function you can use to D.R.Y. this section.
- [ ] revise